SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+## Scope of security vulernabilities
+
+uv is a Python package manager. Due to the design of the Python and its packaging ecosystem, there
+are many cases where uv can execute arbitrary code. For example:
+
+- uv invokes Python interpreters on the system to retrieve metadata
+- uv builds source distributions as described by PEP 517
+- uv may build packages from the requested package indexes
+
```

Comment:
I'm sure I'm missing things here

Reply:
We could perhaps just mention things like `no-build`?

Reply:
That sort of belongs in a "Hardening" document rather than the security policy document?

Maybe once that exists we can just link there?